pkg/operation/seed/seed.go

```diff
@@ -375,8 +375,13 @@ func BootstrapCluster(seed *Seed, secrets map[string]*corev1.Secret, imageVector
 			// Apply status from old ClusterIssuer to retain the issuer's readiness state.
 			new.Object["status"] = old.Object["status"]
 		}
+		vpaRecommendationMerge = func(new, old *unstructured.Unstructured) {
```

Comment:
How about renaming `clusterIssuerMerge` to `retainStatusInformation` and reusing it for `ClusterIssuer` and `VerticalPodAutoscaler`? With that, you can remove `vpaRecommendationMerge`, i.e. avoid duplication.

Reply:
done